.travis.yml

```diff
@@ -18,4 +18,6 @@ deploy:
   on:
     tags: true
 
-script: circuitpython-build-bundles --filename_prefix circuitpython-community-bundle --library_location libraries --library_depth 2
+# Note: if a packaged library is not being included in the bundle, add the folder
```

Comment:
Is there a way we could fail the CI if something is missing? Or could we swap this to a blacklist instead of a whitelist? I'm worried a whitelist will be hard to maintain. Thanks!